src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,16 @@
+public class StringAddCalculator {
+	public static final String MESSAGE_HAS_NEGATIVE_NUMBER = "HAS_NEGATIVE_NUMBER";
+
+	public int splitAndSum(String source) {
+		CalculatorInputString calculatorInputString = new CalculatorInputString(source);
+		if (calculatorInputString.isEmpty())
+			return 0;
```

Comment:
```suggestion
		if (calculatorInputString.isEmpty()) {
				return 0;
		}			
```

자바에선 한 라인도 괄호로 감싸주는 것이 관례입니다 :)

Reply:
넵!!